pages/wishlist/wishlist.html

```diff
@@ -0,0 +1,475 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>FashiFy | Wishlist</title>
+    <link rel="stylesheet" href="../../style.css" />
+  </head>
+  <body>
+    <div class="wishlist-container">
+      <nav class="navigation">
+        <div class="nav-left">
```

Comment:
Having two divs , why is it necessary ? One div can be used.

Reply:
It is possible in wishlist but to maintain the consistency with other pages I have to put nav inside the outer container where for outer container I put height as 100vh